packages/angular/package.json

```diff
@@ -32,7 +32,8 @@
     "prestorybook": "yarn scss && yarn clean:deps && shx cp -r ../../dist/core ../../dist/clr-icons ./dist",
     "storybook": "start-storybook -s .storybook/public -p 6006",
     "prebuild-storybook": "yarn scss && yarn clean:deps && shx cp -r ../../dist/core ../../dist/clr-icons ./dist && yarn predev",
-    "build-storybook": "build-storybook -s .storybook/public .storybook -o ../../dist/website/storybook/angular",
+    "build-storybook": "build-storybook --no-dll -s .storybook/public .storybook -o ../../dist/website/storybook/angular",
+    "postbuild-storybook": "sb extract ../../dist/website/storybook/angular ../../dist/website/storybook/angular/stories.json",
```

Comment:
I'm leaving leaving this sin here for now. It creates a metadata file of all angular stories based on the contents of the app in the above location. We can generate the json file anywhere. 

My initial idea was to use this information to generate <link rel="prerender" http="path/to/story" /> links for all of the stories but with the service worker caching all of the storybook assets (minus the dll javascript) its not used yet. 

Longer term, we will need to build navigation on the demo tab for allowing users to navigate through multiple stories per component and this will give us the information needed to build that out.